lib/Sema/CSSimplify.cpp

```diff
@@ -4954,12 +4954,18 @@ ConstraintSystem::SolutionKind ConstraintSystem::simplifyMemberConstraint(
       performMemberLookup(kind, member, baseTy, functionRefKind, locator,
                           /*includeInaccessibleMembers*/ shouldAttemptFixes());
 
-  auto formUnsolved = [&] {
+  auto formUnsolved = [&](bool activate = false) {
```

Comment:
*Technically* the use of default arguments for a lambda is a C++14 feature, but we can probably get away with it.